core/src/test/scala/org/apache/spark/MapStatusesSerDeserBenchmark.scala

```diff
@@ -123,7 +123,6 @@ object MapStatusesSerDeserBenchmark extends BenchmarkBase {
   }
 
   override def afterAll(): Unit = {
-    tracker.stop()
```

Comment:
Oh, this seems to exist since Apache Spark 3.0.0, doesn't it?
- https://github.com/apache/spark/pull/26169

Reply:
yes, it was suppressed by tryLogNonFatalError